feat[widgets][carousel] : ENG-7291 Improve text for Carousel field

midhunadarvin opened the pull request:
## Description

- update text field in the carousels component
- update dependencies

**Jira**
https://builder-io.atlassian.net/browse/ENG-7291

**Screenshot**
![Screenshot 2024-10-31 at 3 57 46 PM](https://github.com/user-attachments/assets/20a57e5a-05ae-434b-b144-8c801af3d9c6)

samijaber reviewed (approved):
LGTM. @midhunadarvin feel free to push the changes and update the package number in this same PR

midhunadarvin commented:
> LGTM. @midhunadarvin feel free to push the changes and update the package number in this same PR

Updated package number and published the package. Faced issue mentioned here : https://github.com/BuilderIO/builder/issues/3638 , so I have fixed the dependency issue and published again. So there is a jump in the package version from `2.0.0` to `2.0.3`
